server/routes/friendLists/friendLists.route.js

```diff
@@ -0,0 +1,101 @@
+const express = require('express');
+const passport = require('passport');
+const mongoose = require('mongoose');
+const router = express.Router({ mergeParams: true });
```

Comment:
I'm not sure if this is necessary since we don't yet have conflicting param names in the API URL

Reply:
Ohhh true!